test/models/publication_test.rb

```diff
@@ -0,0 +1,81 @@
+require "test_helper"
+require 'pry'
+describe Publication do
+
+  describe 'publication' do
+
```

Comment:
You also need to have a test that a valid publication is valid, maybe using a fixture.